cognite/client/_api_client.py

```diff
@@ -117,13 +56,36 @@ class APIClient:
     }
 
     def __init__(self, config: utils._client_config.ClientConfig, api_version: str = None, cognite_client=None):
-        self._request_session = _REQUESTS_SESSION
-        self._request_session_with_retry = _REQUESTS_SESSION_WITH_RETRY
-
         self._config = config
         self._api_version = api_version
         self._cognite_client = cognite_client
 
+        self._http_client = HTTPClient(
+            session=GLOBAL_REQUEST_SESSION,
+            config=HTTPClientConfig(
+                status_codes_to_retry={429},
```

Comment:
I'm not sure I undrestood the change. Are you retry only HTTP/429?

Reply:
There is noe change here. This client is used for create/update endpoints. It will retry 429's and connection errors. Other endpoints use a http client which is configured to retry 429/502/503, connection errors, and read timeouts.

Reply:
see https://github.com/cognitedata/cognite-sdk-python/pull/726/files/a505e0f8d7eca1c9b74fe7070edd5e9f001b0411#diff-e559e86e19971731b9d9a837e8158c5dR76-R88 and https://github.com/cognitedata/cognite-sdk-python/pull/726/files/a505e0f8d7eca1c9b74fe7070edd5e9f001b0411#diff-e559e86e19971731b9d9a837e8158c5dR128-R130